frontend/src/app/export/export.component.ts

```diff
@@ -10,13 +13,19 @@ import { HttpClient, HttpHeaders } from '@angular/common/http';
 })
 export class ExportComponent {
 
-  constructor(private httpClient: HttpClient) { }
+  config: Config;
+  basePath: string;
+
+  constructor(private httpClient: HttpClient, @Optional()@Inject(BASE_PATH) basePath: string, @Optional() configService: ConfigService) {
+      this.config = configService.getConfig();
```

Comment:
This would cause a null pointer exception if `configService` was null, which is possible from the `@Optional()` annotation.